README.rst

```diff
@@ -46,9 +50,7 @@ Add it using configuration like::
 
   kinto.event_listeners = mp
   kinto.event_listeners.mp.use = kinto_megaphone.listeners
-
-Every listener also needs the following settings (with real values)::
-
   kinto.event_listeners.mp.api_key = foobar
   kinto.event_listeners.mp.url = http://megaphone.example.com/
   kinto.event_listeners.mp.broadcaster_id = remote-settings
+  kinto.event_listeners.mp.match_kinto_changes = /buckets/main /buckets/blocklists /buckets/gfx
```

Comment:
you could add an example where a collection is used.

(and a note that it doesn't work with groups ;))